packages/server/src/routes/grants.js

```diff
@@ -1,6 +1,9 @@
 const express = require('express');
 
 const router = express.Router({ mergeParams: true });
+// TODO: why is this necessary?
+/* eslint-disable import/no-unresolved */
+const { stringify: csvStringify } = require('csv-stringify/sync');
```

Comment:
I just tried it and my eslint is also showing the error. Might be because of csv-stringify being a esm module? `node_modules/csv-stringify/dist/esm/sync` No clue

Reply:
We have this eslint-disable comment in a few other places for imports too. ¯\_(ツ)_/¯